revealable.css

```diff
@@ -0,0 +1,46 @@
+.revealable {
```

Comment:
Is there standard material CSS for this (i.e. are these canonical structural CSS types)?